Group_Constitution

```diff
@@ -0,0 +1,127 @@
+
+4.	Executive List and Duties
+
+The executive committee shall be comprised of three (3) elected officers. These include a President, Secretary, and Treasurer.
+
+The President shall:
+•	Oversee the operations, management and success of the group
+•	Be the spokesperson for the group
+•	Hold signing officer authority along with the Treasurer for financial purposes
+•	Preside over board meetings as well as general meetings
+•	Ensure transition of office to the future Executives
+Additional responsibilities may include: 
+     
+
+
+The Vice-President shall:
+•	Assume duties of the President in his/her absence
+•	Oversee the various committees
+•	Ensure that all the activities of the club meet regulations and policies of the University of Toronto
+•	Coordinate organizational recruitment efforts
+Additional responsibilities may include: 
+     
+
+
+The Secretary shall: 
+•	Make a list of all registered members 
+•	Maintain the web sites and member contact list
+•	Record notes and motions for meetings
+•	Notify all members of meetings
+•	Handle official correspondence of the organization
+Additional responsibilities may include: 
+     
+
+
+
+The Treasurer shall:
+•	Record all financial transactions of the group
+•	Hold signing officer authority along with the President for financial purposes
+•	Maintain a budget of income and expenses along with receipts
+•	Advise members on financial position of the group
+•	Prepare an annual budget for the group as well as budgets for specific events
+Additional responsibilities may include: 
+     
+
+
+
+The group may appoint Directors or Coordinators for various committees such as social committee, publicity committee, and so on; however, such positions do not hold executive decision making authority.
+
+Termination of Executives or General Members:
+
+Any member of the club who commits an act negatively affecting the interests of the club and its members, including non-disclosure of a significant or continuing conflict of interest, may be given notice of removal. The member up for removal shall have the right to defend his/her actions. A vote will be held at an executive meeting, and a two-thirds majority vote of the current executives present in favor of removal is required.  The member must have the right to an appeal before the general membership, and the majority vote of the general membership will have the final say on the matter.
+
+The member will be removed from the club’s membership and will lose any privileges associated with being a member of the club.
+
+Executive members are subject to the same termination or impeachment process and, as determined by the vote, may lose their executive position along with their membership to the group.
+
+
+5.	Elections
+
```

Comment:
It seems so. Maybe not 'elections' per se, but some type of signing officer or executive responsible for the group.

I think June would be better. At least at this point it isn't necessary to complicate things too much! :smiley: